eras/shelley/impl/src/Cardano/Ledger/Shelley/LedgerState/IncrementalStake.hs

```diff
@@ -228,25 +229,19 @@ resolveActiveIncrementalPtrs isActive ptrMap_ (IStake credStake ptrStake) =
 --   UnifiedMap, and the IncrementalStake Only keep the active stake. Active can
```

Comment:
Nitpick: `UMap` here, to be consistent with the code and the other comment?

Reply:
done, thanks.